src/Groups/matrices/MatGrp.jl

```diff
@@ -502,9 +502,11 @@ gen(G::MatrixGroup, i::Int) = gens(G)[i]
 
 ngens(G::MatrixGroup) = length(gens(G))
 
-function order(::Type{T}, G::MatrixGroup) where T <: Union{Integer,fmpz}
-   if get_special(G, :order)==nothing return fmpz(BigInt(GAP.Globals.Order(G.X)))
-   else return T(get_special(G, :order))
+function order(::Type{T}, G::MatrixGroup) where T <: Union{Integer, fmpz}
```

Comment:
An alias for `Union{Integer, fmpz}` might be nice, though; perhaps in Nemo?

Reply:
In Singular.jl, I added `const IntegerLikeTypes = Union{Integer, n_Z, Nemo.fmpz}` some time ago.

Perhaps `const Intiegerish = Union{Base.Integer, Nemo.fmpz}` or `Intlike` or `IntegerLike` or so...?

I mean, it's not *that* bad to type `Union{Base.Integer,fmpz}` but then I sometimes write `Union{fmpz,Base.Integer}` and it gets a bit tedious to grep for it, and I keep finding places where I should use that Union and so a shortcut would be nice ;-).